release_notes.md

```diff
@@ -1,3 +1,78 @@
+**********************************
+# Release Notes for OpenMDAO 3.9.0
+
+April 26, 2021
+
+OpenMDAO 3.9.0 features an API change to the serial/parallel implementation
+in OpenMDAO, a significant performance increase in approximated partials,
+and several other improvements.
+
+Most OpenMDAO components allow inputs and outputs to be added after the
+component is instantiated.  This now applies to ExecComp as well.  New
+expressions can be added to ExecComp using the `add_expr` method.
+
+OpenMDAO can sometimes produce numerous verbose warnings during setup.
+We now implement our own Warning classes so that the verbosity of these
+warnings can be changed through the use of Python's warning filtering
+functionality.
+
+See the following sections for a complete list of changes.
+
+## New Deprecations
+
+- Environment variable OPENMDAO_REQUIRE_MPI is now OPENMDAO_USE_MPI. [#1968](https://github.com/OpenMDAO/OpenMDAO/pull/1968)
+
+## Backwards Incompatible API Changes:
+
+- dynamically shaped connection between distributed outputs and serial inputs are no longer allowed. [#2013](https://github.com/OpenMDAO/OpenMDAO/pull/2013)
+- dynamically shaped connections now by default assume local data transfer to distributed inputs, i.e., size/shape and src_indices in the input correspond to the local part of the output. [#2013](https://github.com/OpenMDAO/OpenMDAO/pull/2013)
+- old behavior of connecting distributed outputs to serial inputs was to have the serial input in each proc contain the full distributed size of the output. To achieve this same behavior, user must now specify src_indices for the input of om.slicer[:] (recommended) or an index array matching the full distrib size of the output. [#2013](https://github.com/OpenMDAO/OpenMDAO/pull/2013)
+
+## Backwards Incompatible NON-API Changes:
+
+- None
+
+## New Features:
+
+- The unit 'as' for attoseconds now works. [#1963](https://github.com/OpenMDAO/OpenMDAO/pull/1963)
+- Added support for sparse FD, setting of jacobian columns, and using sparse arrays in coloring algorithms. [#1967](https://github.com/OpenMDAO/OpenMDAO/pull/1967)
+- Warnings for MPI/PETSc failed import are now suppressed. [#1968](https://github.com/OpenMDAO/OpenMDAO/pull/1968)
+- Added OpenMDAO version to case recorder file [#1971](https://github.com/OpenMDAO/OpenMDAO/pull/1971)
+- Added `add_setup_check()` hook that is called immediately after setup. [#1976](https://github.com/OpenMDAO/OpenMDAO/pull/1976)
+- User can now add expressions to ExecComp using "add_expr". [#1977](https://github.com/OpenMDAO/OpenMDAO/pull/1977)
+- Added argument to list_problem_vars to optionally return unscaled values. [#1980](https://github.com/OpenMDAO/OpenMDAO/pull/1980)
+- Added OpenMDAO-specific warnings. [#1982](https://github.com/OpenMDAO/OpenMDAO/pull/1982)
+- Added notebook utils for upcoming documentation built in Jupyter Book [#1984](https://github.com/OpenMDAO/OpenMDAO/pull/1984)
```

Comment:
not descriptive enough. Is this user facing at all? probably remove

Reply:
It's in miscellaneous because I prefer to include everything in the release notes, even if it doesn't affect most users.  Then there's an easier record of when it was added to the code.